### ISSUE TITLE
line height accessibility issue

### ISSUE THREAD

**Issue by masukomi** [nileane/TangerineUI-for-Mastodon#136]
tl;dr: line height specifications punish users who need large fonts.

## Details
`line-height` is being set in a few places to a specific pixel size. This becomes an increasingly greater accessibility problem as the user's font size exceeds the size defined in the CSS. To put it another way, the more help a user needs from large fonts the more the CSS punishes them for needing help. I'm currently running at 24 with with minimum ALSO set to 24.  Not huge, but large enough to see the problem.

As a general rule font size, leading, and kerning should _never_ be specified with pixel dimensions, but instead using percentages or `rem`  / `em` values. 

## Screenshots:
This first screenshot shows what it looks like for me with my large fonts. Note that the lines are starting to mush together.
![CleanShot 2024-12-13 at 12 29 20@2x](https://github.com/user-attachments/assets/73c867b6-c19a-4582-99b5-58a592d8a031)

This second screenshot shows what happens when i set `line-height: normal !important`  This probably isn't the _right_ solution. I've probably broken something I haven't noticed yet, but it's a solution that seems to work.

![CleanShot 2024-12-13 at 12 29 10@2x](https://github.com/user-attachments/assets/b425c968-f0b1-48c9-9147-37f693ab931a)

## Proposal
1. get rid of `line-height` specification wherever it isn't absolutely necessary. 
2. change any that remain to be defined in `rem` or as percentages.

**Comment by nileane** [comment]
Hi! Thanks for reporting this issue. I won't have time to address this before the holidays, but I just wanted to drop in and write that I agree with you and will be fixing this :)

**Comment by Artusamak** [comment]
Here is a PR to try to fix this issue. :)

**Comment by nileane** [comment]
#143 Merged!
Thanks again y'all!
